packages/x-components/src/x-modules/tagging/service/__tests__/session.service.spec.ts

```diff
@@ -1,46 +1,38 @@
+import { InMemoryStorageService } from '@empathyco/x-storage-service';
 import { DefaultSessionService } from '../session.service';
-import { MockedStorageService } from './utils/mock-storage.service';
 
 describe('testing session id service', () => {
-  const prefix = 'test';
-  const mockedStorageService = new MockedStorageService(prefix);
+  const mockedStorageService = new InMemoryStorageService();
   const sessionService = new DefaultSessionService(mockedStorageService, 1);
```

Comment:
I don't think this way of testing gives us enough confidence. This test is reusing the same `InMemoryStorageService` for all tests, without resetting it.

Reply:
reseting it in the beforeEach